### ISSUE TITLE
随机种子设置

### ISSUE THREAD

**Issue by hxdtest** [hxdtest/torch_best_practice#1]
import torch 
seed = 42
torch.manual_seed(seed)

rng_state = torch.load("data.pt")
torch.set_rng_state(rng_state)
# 执行一些随机操作
a = torch.randn(3)
print(a)

# 获取当前RNG状态
rng_state = torch.get_rng_state()

torch.save(rng_state,"data.pt")

# 在此基础上执行更多随机操作
b = torch.randn(3)
print(b)

# 假设我们希望从上次停止的地方继续
torch.set_rng_state(rng_state)
c = torch.randn(3)
print(c)  # 这将产生与变量b相同的张量

仅仅设计随机种子可以保证每次产生的随机数相同，但是为了保持状态，需要保存cpu/gpu的随机状态。 在设置完成随机种子后，加载随机状态，此处需要补充一个测试用例。
